src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {

```

Comment:
You do not need to extend ColorSupplier class here.
If you want to use getRandomColor() method you can simply create new instance of ColorSupplier class